airbyte-webapp/src/packages/cloud/views/users/AccountSettingsView/components/NameSection/NameSection.tsx

```diff
@@ -0,0 +1,59 @@
+import { Field, FieldProps, Form, Formik } from "formik";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import { LoadingButton } from "components";
+import { LabeledInput } from "components/LabeledInput";
+
+import { RowFieldItem } from "packages/cloud/views/auth/components/FormComponents";
+import { Content, SettingsCard } from "pages/SettingsPage/pages/SettingsComponents";
+
+import { useName } from "./hooks";
+import { useCurrentUser } from "packages/cloud/services/auth/AuthService";
+import FeedbackBlock from "pages/SettingsPage/components/FeedbackBlock";
+
+const NameSection: React.FC = () => {
```

Comment:
We're moving away from exporting defaults, so this can be:

```suggestion
export const NameSection: React.FC = () => {
```

The same suggestion applies for other areas of the code, then you can just export it directly on the `index.ts` file.